client/ayon_resolve/plugins/publish/extract_editorial_package.py

```diff
@@ -0,0 +1,190 @@
+from pathlib import Path
+
+import pyblish.api
+import opentimelineio as otio
+
+from ayon_core.pipeline import publish
+from ayon_resolve.api.lib import (
+    maintain_current_timeline,
+    get_current_project,
+    export_timeline_otio_native
+)
+from ayon_resolve.otio import davinci_export
+
+from ayon_resolve.api import bmdvr
+
+
+class ExtractEditorialPackage(publish.Extractor):
+    """
+    Extract and Render intermediate file for Editorial Package
+
+    TODO:
+        - find reference video representation
+        - figure out target publishing rootless path for reference paths
+        - swap all references file names in OTIO file point at the rootless path
+    """
+
+    label = "Extract Editorial Package"
+    order = pyblish.api.ExtractorOrder + 0.45
+    families = ["editorial_pkg"]
```

Comment:
I tried to publish/load an `editorial_pkg` but unfortunately it did not work for me straight on.
![resolve_failed_integration](https://github.com/user-attachments/assets/34c87b03-c2a6-4bf6-9cad-f22cc1281555)

Investigating a bit, I realized neither `Extract Editorial Package` nor `Extract Intermediate Representation` were part of my plugins loop.
![resolve_missing_plugins](https://github.com/user-attachments/assets/f7ed6f31-cdf6-4f5f-adb1-256ec16c678f)

Investigating a bit, I realized a `families` mismatch. The instance data I got out of the creator is:
```
Editorial Package: {'family': 'default', 'name': 'TimelineImg', 'id': 'pyblish.avalon.instance', 'productType': 'editorial_pkg', 'folderPath': '/Root', 'productName': 'editorial_pkgMain', 'active': True, 'variant': 'Main', 'taskId': '415050805f1111ef93b231cb88eabe44', 'taskName': 'Generic', 'mediaPoolItem': <BlackmagicFusion.PyRemoteObject object at 0x00000000054DAE70>, 'taskEntity': {'attrib': {'resolutionHeight': 1080, 'frameEnd': 1001, 'resolutionWidth': 1920, 'handleStart': 0, 'endDate': None, 'clipOut': 1, 'frameStart': 1001, 'tools': None, 'pixelAspect': 1.0, 'clipIn': 1, 'handleEnd': 0, 'fps': 25.0, 'startDate': None, 'description': None}, 'active': True, 'taskType': 'Generic', 'id': '415050805f1111ef93b231cb88eabe44', 'name': 'Generic', 'assignees': [], 'status': 'Not ready', 'folderId': 'c73a0ac05f1011ef93b231cb88eabe44', 'label': 'Generic', 'data': None, 'tags': []}, 'task': 'Generic'}
```
I tweaked `families = ["default"]` for both and it properly went through. But I'm not sure this is the proper fix.

Reply:
Hey @robin-input it seems that the publishable instance was not properly created via Creator. Can you confirm please?

Reply:
I have tested the workflow and it works for me 
![image](https://github.com/user-attachments/assets/d94f13be-1923-4ad3-9f89-555f3f612295)

The timeline was manually created and then selected and Created/converted into publishable instance. I will update the description so `Testing steps` are included